### PR TITLE
Shaker page

### PR DESCRIPTION
**Pull request by ntsyhaniuk**
base functionality

### CODE REVIEW

File: src/app/make-cocktail/make-cocktail.component.scss [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+      width: 100%;
+      button {
+        margin: 10px 0;
+      }
+    }
+  }
+}
+
+@keyframes spaceboots {
+  0% { -webkit-transform: translate(6px, 3px) rotate(0deg); }
```
Comment (skrechko) [inline src/app/make-cocktail/make-cocktail.component.scss]: Just for the future, we have postprefixer inside of angular-cli, no need to apply vendor prefixes (-webkit-, -moz-, -ms) at all.
Us just `transform: translate(-3px, -6px) rotate(-3deg);`

File: src/app/make-cocktail/make-cocktail.component.scss [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+      justify-content: space-around;
+      width: 100%;
+      button {
+        margin: 10px 0;
+      }
+    }
+  }
+}
+
+@keyframes spaceboots {
```
Comment (skrechko) [inline src/app/make-cocktail/make-cocktail.component.scss]: Better to use custom prefixes for custom animation for instance:
spaceboots --> ff-space-boots or eg-space-boots

File: src/app/make-cocktail/make-cocktail.component.scss (hunk 1 above)
Comment (skrechko) [reply src/app/make-cocktail/make-cocktail.component.scss]: pref

**Review by skrechko — APPROVED** [review]

File: src/app/make-cocktail/make-cocktail.component.scss (hunk 1 above)
Comment (ntsyhaniuk) [reply src/app/make-cocktail/make-cocktail.component.scss]: My apologies, it was copy-paste 😀

**Review by khrystyna-pylypchuk — APPROVED** [review]

**Pull request merged**